lib/mix/tasks/phoenix.new.ex

```diff
@@ -33,6 +34,12 @@ defmodule Mix.Tasks.Phoenix.New do
 
     copy_from template_dir, path, application_name, &EEx.eval_file(&1, binding)
     copy_from static_dir, Path.join(path, "priv/static"), application_name, &File.read!(&1)
```

Comment:
Well, you do want to gitignore priv static when using a build tool, so we need to lookup for alternatives. We will also need to move the phoenix logo to web/static/assets.